sdk/core/azure-core/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Breaking Changes
 
 - `Azure::Core::Http::Url::AppendPath` now does not encode the input by default.
+- `azure.hpp` renamed to `strings.hpp`
+- `Azure::Core::Details::LocaleInvariantCaseInsensitiveEqual` moved to `Azure::Core::Strings::LocaleInvariantCaseInsensitiveEqual`
+- `Azure::Core::Details::ToLower` moved to `Azure::Core::Strings::ToLower`
```

Comment:
This isn't a breaking change. We are making a private method public. This should go under new features.

Reply:
Things in ::Details namespaces aren't private.  They are simply methods/details which have an implicit "do not use" contract.

Reply:
New feature makes sense to me as well